hugegraph-api/src/main/java/org/apache/hugegraph/api/traversers/CrosspointsAPI.java

```diff
@@ -84,6 +85,9 @@ public String get(@Context GraphManager manager,
                                                       dir, edgeLabel, depth,
                                                       maxDegree, capacity,
                                                       limit);
-        return manager.serializer(g).writePaths("crosspoints", paths, true);
+        measure.addIterCount(traverser.vertexIterCounter.get(),
+                             traverser.edgeIterCounter.get());
+        return manager.serializer(g, measure.getResult())
```

Comment:
prefer to update the `serializer(g)` method and set default an ApiMeasure

Reply:
Rejected: 
1. `ApiMeasurer` should be passed in from an outside api implementation so that it can carry data
2. not all apis need ApiMeasurer and keeping the ApiMeasurer null for those APIs can prevent `ApiMeasurer` from being serialized